core/src/main/java/org/elasticsearch/indices/ttl/IndicesTTLService.java

```diff
@@ -340,7 +340,7 @@ public void await() {
             try {
                 condition.await(timeout.millis(), TimeUnit.MILLISECONDS);
             } catch (InterruptedException e) {
-                Thread.interrupted();
+                Thread.currentThread().interrupt();
```

Comment:
Looking what this does, an interruption is ignored and will only used to cause an extra TTL check or shutdown. I think doing nothing is the right move here.